content/en/platform/corda/5.2/developing-applications/api/ledger/utxo-ledger/vault-queries.md

```diff
@@ -278,7 +285,7 @@ This transforms each element to a `String` object, which is the given state’s
 #### Collecting
 
 Collecting is used to collect results set into one single integer.
-For this, implement the `net.corda.v5.ledger.utxo.query.VaultNamedQueryCollector<R, T>` interface. The `<R>` parameter is the type of the original result set (in this case `String` because of transformation) and `<T>` is the type collected into (in this can an `Int`).
+For this, implement the `net.corda.v5.ledger.utxo.query.VaultNamedQueryCollector<R, T>` interface. The `<R>` parameter is the type of the original result set (in this case `String` because of transformation) and `<T>` is the type collected into (in this case, an `Int`).
```

Comment:
```suggestion
For this, implement the `net.corda.v5.ledger.utxo.query.VaultNamedQueryCollector<R, T>` interface. The `<R>` parameter is the type of the original result set (in this case `String` because of transformation) and `<T>` is the type collected into (in this case, an `Int`).
```
R and T listed as an unordered list?

Reply:
Updated